Vwitter/View Controllers/SelectAudienceViewController.m

```diff
@@ -165,7 +165,7 @@ - (IBAction)didTapVent:(id)sender {
     
     __weak typeof(self) weakSelf = self;
     [PFCloud callFunctionInBackground:@"postVent"
-                       withParameters:@{@"currentUserId":[VWUser currentUser].objectId, @"ventContent":self.ventContent, @"ventAudiencesIds":ventAudiencesIds}
+                       withParameters:@{@"currentUserId":[VWUser currentUser].objectId, @"ventContent":self.ventContent, @"ventAudiencesIds":ventAudiencesIds, @"selectedTrackUri":self.selectedTrack.uriString, @"startTimestamp":self.selectedTrack.startTimestamp, @"endTimestamp":self.selectedTrack.endTimestamp}
```

Comment:
same here, this is dangerous we really need to be doing validation on some of these fields before inserting them in a dictionary sending them to the backend